### PR TITLE
Additional icons - Part 2

### PR DESCRIPTION
**Pull request by ziegenberg**
Adds icons for eveonline (#450), oculus (#536), pushover (#540), anonaddy (#473), livelyme (#472), coinjar (#478), starcitizen (#483), pretix (#503), figma (#511), VEXXHOST (#484), Greenhost (#515), Lobsters (#523), mongodb (#318)

### CODE REVIEW

**Comment by ziegenberg** [comment]
This is also now ready for review. If this makes it into 0.7.1 it will leave only a few thumbnail requests open and almost halve the number of open tickets.
If #491 is merged first, I can rebase this PR onto master again to prepare for merging.

File: app/src/main/res/drawable/thumb_figma.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="295.49dp"
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_figma.xml:3]: Would be best to keep under 256x256

Comment (ziegenberg) [reply app/src/main/res/drawable/thumb_figma.xml:3]: oops, that one slipped through. I'll fix it in an instant

Comment (ziegenberg) [reply app/src/main/res/drawable/thumb_figma.xml:3]: https://developer.android.com/studio/write/vector-asset-studio#when even recommends 200dp x 200dp

**Pull request merged**